src/main/java/org/fundacionjala/coding/SorterInnerContent.java

```diff
@@ -0,0 +1,37 @@
+package org.fundacionjala.coding;
+
+public class SorterInnerContent {
+    public static final int MINIMUM_LENGHT_OF_WORD = 3;
+
+    public String sortInnerContent(final String text) {
+        String result = "";
+        String[] words = text.split(" ");
+        for (int i = 0; i < words.length; i++) {
+            int wordLength = words[i].length();
+            if (wordLength > MINIMUM_LENGHT_OF_WORD) {
+                StringBuilder innerWord = new StringBuilder(words[i].substring(1, wordLength - 1));
+                innerWord = sortDescending(innerWord);
+                words[i] = innerWord.append(words[i].charAt(wordLength - 1)).insert(0, words[i].charAt(0)).toString();
+            }
+        }
+        result = String.join(" ", words);
+        return result;
+    }
+
+    private StringBuilder sortDescending(final StringBuilder text) {
```

Comment:
This method is no longer necessary.
You can use the Arrays.sort already defined on Java